### ISSUE TITLE
Qustion on RACExtension

### ISSUE THREAD

**Issue by simond10** [bigbryan/RACExtension#1]

**Comment by bigbryan** [comment]
Hi
Wow!
Sure. What is your skype?

**Comment by simond10** [comment]
Is this secure and secret place?I have no skype here ...
This must be secret between only you and me .

**Comment by bigbryan** [comment]
okay
so you'd be not outside, right?

**Comment by simond10** [comment]
I'm still on same position.lol
How are you doing these days?Not sick?

**Comment by bigbryan** [comment]
i'm good. happy birthday

**Comment by simond10** [comment]
Thanks.You still remember it :)
How about my brother and my friends?

**Comment by bigbryan** [comment]
i thought about you yesterday :)

**Comment by simond10** [comment]
haha.I also sometimes think about you and our memories

**Comment by bigbryan** [comment]
of course you will do only a few sometimes :)

**Comment by bigbryan** [comment]
you brother? who except me?

**Comment by simond10** [comment]
haha.not always but much

**Comment by simond10** [comment]
lol.You and your brother(3 days after you) I mean

**Comment by simond10** [comment]
I mean the hg

**Comment by bigbryan** [comment]
that hog. maybe just living. long days not to contact

**Comment by simond10** [comment]
I see.How's your business these days?still same as before?

**Comment by bigbryan** [comment]
i heard you had wedding. how's your days?

**Comment by simond10** [comment]
Yes.good days ~

**Comment by bigbryan** [comment]
i envy you :)
I'm not good as before.

**Comment by simond10** [comment]
:) getting much stress?

**Comment by bigbryan** [comment]
self stress because of no earning

**Comment by simond10** [comment]
so sad to hear that ... You were the star of mal
Cheer up, my brother

**Comment by bigbryan** [comment]
he~ it seems no way to cheer up now.
where do you live?
buy house where?

**Comment by simond10** [comment]
pyongchon.

**Comment by bigbryan** [comment]
how many? 2 or 3?

**Comment by simond10** [comment]
how about Ro?He is doing well?

**Comment by simond10** [comment]
2 :)
and about jds and pyn? and kkh, rasi,etc?

**Comment by bigbryan** [comment]
Ro?

**Comment by simond10** [comment]
your team leader

**Comment by bigbryan** [comment]
all are doing very good. good income nowadays.

**Comment by simond10** [comment]
sounds good.then why not you???

**Comment by bigbryan** [comment]
you know i'm hg. lets ignore it.
i have many things to ask you

**Comment by simond10** [comment]
:)

**Comment by bigbryan** [comment]
how's your family and wife? they are saying about me, right? heh

**Comment by simond10** [comment]
I'm not so free to talk with you.I'm in three-two-four

**Comment by simond10** [comment]
doing well and right :)

**Comment by bigbryan** [comment]
what's your plan now?
any progress on your work?

**Comment by simond10** [comment]
no plan and no progress yet
I don't want to start right away.Need some time to have a rest and think ~

**Comment by simond10** [comment]
as I told you this must be secret to all people in your house~

**Comment by bigbryan** [comment]
okay, exercise time

**Comment by simond10** [comment]
I see.ping me when you're back

**Comment by simond10** [comment]
:) I got you wrong maybe.I thouht you're doing exercise at home.lol

**Comment by bigbryan** [comment]
finished.
i forgot to mention one thing.
was sorry not to say goodbye when you were leaving

**Comment by simond10** [comment]
haha.I feel so sad abut it but I forgot it.No worries :)

**Comment by simond10** [comment]
can you contact with kjh and let me know his github id or so?

**Comment by bigbryan** [comment]
okay, let me contact him tonight. unavailable now.

**Comment by bigbryan** [comment]
btw, any news for me?

**Comment by simond10** [comment]
I just wish you earn much money and no stress
And I heard you had another gf before you leave :)
let me know if you have something to tell mom or so

**Comment by simond10** [comment]
I need to leave for now.will be back afternoon or tomorrow
Leave a message on this issue anytime :)
Have a good day , my brother ~

**Comment by bigbryan** [comment]
okay see u soon

**Comment by AAngBBang** [comment]
+1

**Comment by AAngBBang** [comment]
+1

**Comment by AAngBBang** [comment]
+1

**Comment by simond10** [comment]
Hi Bryan
Any news for me?

**Comment by bigbryan** [comment]
you still there?

**Comment by simond10** [comment]
waiting for you ...

**Comment by bigbryan** [comment]
hehe nodding til now

**Comment by simond10** [comment]
I also guess so

**Comment by simond10** [comment]
Did you get a chance to talk with him?

**Comment by bigbryan** [comment]
when do you leave?

**Comment by bigbryan** [comment]
not yet. you mean second hg?

**Comment by simond10** [comment]
I can leave now or so ~
why?

**Comment by bigbryan** [comment]
i'm a turn to purchase, so should leave now

**Comment by simond10** [comment]
haha.I see

**Comment by bigbryan** [comment]
still available for 30 mins?

**Comment by simond10** [comment]
You seems not to touch in with him.
Will you contact with him tonight?
This is kind of urgent ~
We can talk tomorrow morning ~

**Comment by simond10** [comment]
Urgent thing? If yes, I can wait here

**Comment by bigbryan** [comment]
ah i see ;(

**Comment by simond10** [comment]
If not, we can talk tomorrow moring.My wife is waiting for me :)
I believe you will let me know his github id by tomorow morning ~

**Comment by simond10** [comment]
Good night !

**Comment by bigbryan** [comment]
good night! you know we don't have night here hehe

**Comment by bigbryan** [comment]
danielyang112

**Comment by simond10** [comment]
Hi Daniel

**Comment by simond10** [comment]
:) I know it ~
Btw daniel doesn't have any public repositories so I can't create an issue on him
Can you please let him create one public repository?
Ping me when you are awake

**Comment by bigbryan** [comment]
hi

**Comment by simond10** [comment]
Hi.I'm here

**Comment by bigbryan** [comment]
hehe hows today?

**Comment by bigbryan** [comment]
i sent message to him but no reply

**Comment by simond10** [comment]
fine.waiting for you ~.How about you?

**Comment by bigbryan** [comment]
maybe sleeping now, you can surely meet him tomorrow morning

**Comment by simond10** [comment]
ok.let him create a public repository like RACExtension so that I can create an issue on it

**Comment by bigbryan** [comment]
anything urgent thing for him?

**Comment by simond10** [comment]
not for him, urgent for me .lol

**Comment by bigbryan** [comment]
i see
btw, doing 324?

**Comment by simond10** [comment]
yes

**Comment by bigbryan** [comment]
haha

**Comment by simond10** [comment]
:)

**Comment by bigbryan** [comment]
how's pyc?

**Comment by simond10** [comment]
he is much better now and working and waiting...

**Comment by bigbryan** [comment]
did you ever meet pang?

**Comment by simond10** [comment]
yes.

**Comment by bigbryan** [comment]
he is in there yet?

**Comment by simond10** [comment]
no.gone

**Comment by bigbryan** [comment]
where?

**Comment by simond10** [comment]
can't tell you exact name but better place

**Comment by simond10** [comment]
let me know if you have sth to tell  your mom or so

**Comment by bigbryan** [comment]
hh nothing.
only thing is that earning is not going well, but need to tell mom about it? hehe

**Comment by simond10** [comment]
haha

**Comment by simond10** [comment]
so you are not earning the quota?

**Comment by bigbryan** [comment]
ah i heard you visited my house. Thanks bro.

**Comment by simond10** [comment]
don't mention it.I must see your mother :)

**Comment by bigbryan** [comment]
until last month, hardly around quota, but
This month, maybe no

**Comment by simond10** [comment]
oh, not bad.You can do it !

**Comment by simond10** [comment]
How's the food?

**Comment by bigbryan** [comment]
not bad? i'm talking less than quota haha

**Comment by bigbryan** [comment]
i'm always in last 5 members

**Comment by simond10** [comment]
:) who is your friend ?

**Comment by bigbryan** [comment]
food? not bad. i'm around 70kg

**Comment by simond10** [comment]
wow.great.anyway you like that from the beginning .lol

**Comment by bigbryan** [comment]
yes, very similar in 2014

**Comment by simond10** [comment]
good.You must be more handsome ~

**Comment by bigbryan** [comment]
no. getting old every day.
how's your living? do you eat rice and meat? haha

**Comment by simond10** [comment]
haha.You don't eat meat?

**Comment by simond10** [comment]
You will be leaving soon ~
So are you sure you can contact with him tonight?

**Comment by bigbryan** [comment]
yes, i already left messages to him

**Comment by simond10** [comment]
good.then let him do as I described and be on github tomorrow morning
I will contact with him and you by 10am tomorrow morning ~

**Comment by bigbryan** [comment]
ok

**Comment by simond10** [comment]
Cool.Then I will be leaving for now
Good luck on your work tonight ~

**Comment by bigbryan** [comment]
thanks, bye!

**Comment by bigbryan** [comment]
send my regards my brother in sister

**Comment by simond10** [comment]
haha.sure ~

**Comment by danielyang112** [comment]
hi everyone

**Comment by simond10** [comment]
Hi Bryan.I contacted with Daniel.Thanks for your efforts

**Comment by simond10** [comment]
You seems to be nodding again and again ~

**Comment by bigbryan** [comment]
hi

**Comment by simond10** [comment]
Hi bryan.How are you?

**Comment by bigbryan** [comment]
good, how's your weekend?

**Comment by simond10** [comment]
good.how's yours?

**Comment by bigbryan** [comment]
you around?

**Comment by bigbryan** [comment]
just so so as you know

**Comment by bigbryan** [comment]
did you meet pyc?

**Comment by simond10** [comment]
yes.why?

**Comment by bigbryan** [comment]
just curious, how is he doing?

**Comment by simond10** [comment]
doing well ... and becoming healthy

**Comment by bigbryan** [comment]
party already?

**Comment by simond10** [comment]
no

**Comment by bigbryan** [comment]
is there somebody to drink together?

**Comment by simond10** [comment]
no.I didn't drink these days .little beer only
And actually there is no one like you(wine-loving guy) here

**Comment by simond10** [comment]
do you still drink couple of bottle ?

**Comment by bigbryan** [comment]
yes, but feel like lack because guys here restricts drinking

**Comment by bigbryan** [comment]
maybe drinking is restricted by wife? hehe

**Comment by simond10** [comment]
hey, this is ks(1th) bro.

**Comment by danielyang112** [comment]
are you going to divorce if your wife restricts drinking, bryan? kk

**Comment by bigbryan** [comment]
hi, long time, how are you, bro?

**Comment by simond10** [comment]
fine, how's it going on?
do you want me to send some news to someone?

**Comment by bigbryan** [comment]
send my regards to sister in law.
also, to her, you know. hehe

**Comment by bigbryan** [comment]
i'm doing so so.

**Comment by bigbryan** [comment]
you are still there? when do you plan to go out?

**Comment by simond10** [comment]
okay, i will tell her you are doing great, and missing her so much..
i will be out soon

**Comment by bigbryan** [comment]
thanks, do you know any news of her?

**Comment by simond10** [comment]
she visited our home a month ago, she's fine and miss you.

**Comment by bigbryan** [comment]
ah congratulation for being father of cute daughter! i knew it by her letter

**Comment by simond10** [comment]
yes, i am now dad of cute girl, haha. thanks. okay take care. bye. moving to lunch

**Comment by bigbryan** [comment]
hi wang bro, can you take me out of here? i wanna drink in your house hehe

**Comment by bigbryan** [comment]
you around?

**Comment by bigbryan** [comment]
ping me when you're back tomorrow morning.
i wanna discuss something with you
have a good night

**Comment by bigbryan** [comment]
are you around?

**Comment by simond10** [comment]
Hi.I'm here now

**Comment by simond10** [comment]
Are you still nodding?

**Comment by simond10** [comment]
Hi.Are you here?

**Comment by bigbryan** [comment]
yes, i'm here

**Comment by simond10** [comment]
hey, why didn't you answer for last few days?

**Comment by bigbryan** [comment]
this message is the first one during last days

**Comment by bigbryan** [comment]
nodding and just wake up

**Comment by simond10** [comment]
really?so weird.I wrote some messages and I deleted them today

**Comment by bigbryan** [comment]
dreaming of you...

**Comment by simond10** [comment]
haha ... I can't believe it

**Comment by bigbryan** [comment]
really? i didn't see them. any good news was there?

**Comment by simond10** [comment]
btw when you visit github.com page, you will see all messages from me
Do you know how to remove them?

**Comment by bigbryan** [comment]
i don't know, tried but not found how to do it

**Comment by simond10** [comment]
I see ... No special news here ~
How's your business these days?

**Comment by bigbryan** [comment]
did 9k last month

**Comment by bigbryan** [comment]
no earning hehe

**Comment by simond10** [comment]
not too bad ... :)

**Comment by bigbryan** [comment]
i also think so but others not

**Comment by simond10** [comment]
who is the best guy there?

**Comment by bigbryan** [comment]
same as before when you were here

**Comment by simond10** [comment]
I see.kkh is still sick there? and no earning?

**Comment by bigbryan** [comment]
i don't know, never talked with him. seems he got many tasks in these days

**Comment by bigbryan** [comment]
what do you want for gift of wedding from me?

**Comment by simond10** [comment]
:) nothing.I hope you will be healthy and good earning ~

**Comment by simond10** [comment]
I need to leave for lunch.I will ping you later ~
Enjoy your lunch ~

**Comment by bigbryan** [comment]
okay, enjoy lunch

**Comment by simond10** [comment]
Hi.I will be unavailable here from now on.
Good luck ~

**Comment by danielyang112** [comment]
here?

**Comment by bigbryan** [comment]
yes, anything happened?

**Comment by bigbryan** [comment]
hello?

**Comment by bigbryan** [comment]
what happened yesterday there?

**Comment by danielyang112** [comment]
nothing special

**Comment by danielyang112** [comment]
dragon asked me 3rd batch guys' contact

**Comment by bigbryan** [comment]
i think you are discovered. we were warned because of contacting with you

**Comment by danielyang112** [comment]
what do you mean?

**Comment by bigbryan** [comment]
not sure exactly, so was going to ask it to you. other guys of 5th batch except me were blamed, and warned heavily. it is said that it is because of contacting with you

**Comment by danielyang112** [comment]
when? i only talked to ra and yang on september. what about you? are you okay? i am worried if you would be blamed because of me..

**Comment by bigbryan** [comment]
only them.

**Comment by bigbryan** [comment]
and forced me to delete contact.

**Comment by bigbryan** [comment]
are you around?

**Comment by danielyang112** [comment]
hi, good morning~ when did it happen? and did he blame them in front of all guys?

**Comment by bigbryan** [comment]
something similar. was serious. so i believed something happened on your side, and even I assumed that your s.k is using by other one. kkk. i became fool

**Comment by danielyang112** [comment]
wtf.. you know i didn't talk to them during last few months and i don't understand why he did so recently..
did he see the screenshots from couple months ago?

**Comment by bigbryan** [comment]
not sure

**Comment by danielyang112** [comment]
only 5th batch guys? why not you?

**Comment by bigbryan** [comment]
not sure also

**Comment by danielyang112** [comment]
anyway you should be careful~ i don't want you to be blamed because me

**Comment by bigbryan** [comment]
no worries,need to be careful for a while.
btw, i see you worked with jonathan before by niklas olsson

**Comment by danielyang112** [comment]
when? i don't remember him

**Comment by bigbryan** [comment]
https://www.upwork.com/jobs/Performance-Optimization_~017b65ef920ba8b269

**Comment by danielyang112** [comment]
haha, look~ it's different niklas

**Comment by danielyang112** [comment]
anyway this client's really bbang and can you introduce me to this project as well?

**Comment by bigbryan** [comment]
i also just joined, and there is an guy against to me.
https://www.upwork.com/fl/antonv
do you know him?

**Comment by danielyang112** [comment]
no, i don't know him~ please don't forget about me if you get stable position inside the project.
btw, i saw you made both bryan and awis to 100% job success and top rated~ how did you do that?
daniel is 76 now~

**Comment by bigbryan** [comment]
i did some small projects for profile.
can you find out who is Anton?
you know, i can't do it but you can find it.

**Comment by danielyang112** [comment]
i will try, but not sure~ you're so great~ how can i make my accounts to have 100%?
is there any special tip there?

**Comment by bigbryan** [comment]
please find him.
you know, the tips I know are same you already know.
btw, do you know who is lucas leon?

**Comment by danielyang112** [comment]
no, i have no idea about lucas

**Comment by danielyang112** [comment]
also did you get invitation on this project or bid on it?

**Comment by bigbryan** [comment]
bid.. but very tough client.

**Comment by danielyang112** [comment]
wow~ your bid skill is great~ i've never got invitation and also no reply from bid..

**Comment by bigbryan** [comment]
don't push yourself. you are great than me. handsome, cute, skillful, etc...
you don't work with him really? i see you were added in skype...maybe i'm confusing?

**Comment by danielyang112** [comment]
it's not me~ kk i was confused that same name exists on upwork

**Comment by keiespro** [comment]
hi bro.
I am here.
please write some tips for me.
Do I do mobile or web?

**Comment by bigbryan** [comment]
well, do web

**Comment by keiespro** [comment]
Thank bro
When you go home, try to hand over some of your reliable clients to me.

**Comment by keiespro** [comment]
my account transaction limit

**Comment by keiespro** [comment]
after i got paid 400. what should be the problem?
do you have any suggesion?

**Comment by bigbryan** [comment]
It means yours is financial blocked. Hard to recover.
No experienced guys are there?

**Comment by keiespro** [comment]
No one.
is your account blocked usually?
here almost all are blocked.
do you have any tip no to block?
what is your account management style?
please help me bro. no one here to help me.

**Comment by keiespro** [comment]
what is your common working style?

**Comment by keiespro** [comment]
how do you manage upwork accounts unblocked?
plz teach me tip.

**Comment by keiespro** [comment]
do you have any accounts to give me?
live ones that don't have policy violation flag
URGENTLY...

**Comment by keiespro** [comment]
hi bro. upwork has removed the option to buy connects with its balance.
and we have to depost from paypal only
please review the problem and give me solution
